lib/utils/utils.go

```diff
@@ -322,6 +323,31 @@ func IsValidHostname(hostname string) bool {
 	return true
 }
 
+// IsValidUnixUser checks if a string represents a valid
+// UNIX username.
+func IsValidUnixUser(u string) bool {
```

Comment:
Are the tests in the source repo? Could we add a few here?

Reply:
Tests? We didn't write tests 7 years ago! 😂